kalkulator.py

```diff
@@ -3,7 +3,11 @@ def get_info():
 def dodaj(a, b):
 	wynik=a+b
 	return wynik
+def odejmij(a, b):
+	return a-b
```

Comment:
Tutaj lepiej rozbić to na dwie linijki tak jak w dodawaniu, wprowadzając zmienną wynik=a-b